salt/backups/templates/backup_mysql.sh

```diff
@@ -17,7 +17,8 @@ PASSPHRASE={{ settings.duplicity_passphrase }} /usr/bin/duplicity \
           --s3-use-server-side-encryption {{ backupdir }} \
           --archive-dir {{ cachedir }} \
           --full-if-older-than 1W \
-          --allow-source-mismatch --tempdir /backups \
-          s3+http://odl-operations-backups/{{ settings.get('directory', 'mysql') }}/
+          --allow-source-mismatch --tempdir /backups/tmp/ \
+          s3+http://odl-operations-backups/{{ settings.get('directory', 'mysql') }}/ \
+          {{ settings.database }}/
```

Comment:
My thinking is to map each database (edxapp, xqueue) separately in case we want to backup/restore one or the other separately.